Assets/HoloToolkit/Utilities/Scripts/Editor/SolverHandlerEditor.cs

```diff
@@ -12,24 +12,45 @@ public class SolverHandlerEditor : ControllerFinderEditor
     {
         private SerializedProperty trackedObjectReferenceProperty;
         private SerializedProperty transformTargetProperty;
+        private SerializedProperty additionalOffsetProperty;
+        private SerializedProperty additionalRotationProperty;
+        SolverHandler solverHandler;
 
         protected override void OnEnable()
         {
             base.OnEnable();
 
             trackedObjectReferenceProperty = serializedObject.FindProperty("trackedObjectToReference");
             transformTargetProperty = serializedObject.FindProperty("transformTarget");
+            additionalOffsetProperty = serializedObject.FindProperty("additionalOffset");
+            additionalRotationProperty = serializedObject.FindProperty("additionalRotation");
+
+            solverHandler = target as SolverHandler;
```

Comment:
safe cast target to solver handler.

Reply:
Sorry, I'm not sure I understand your feedback.

Reply:
`solverHandler = (SolverHandler)target;`

Reply:
actually, as is the "safer" cast in that it ensures that the cast does not throw in the cases where it is invalid. I would much rather check for a null return than have an exception handler.

Reply:
It won't ever not be that type anyway, but everywhere else in code it's written the other way.

So either we stick to the established pattern or update it everywhere in all our other editor inspectors.

Reply:
I agree we should be consistent. ever since the as keyword was added to C#, I have preferred to use it over a standard cast syntax… except with non-nullable value types (where it cannot be used). exceptions caused by bad casts are some of my least favorite ones to encounter.

I am currently in the process of updating the vNext coding guidelines and am happy to add a section on how to cast types.

I am also happy to take on updating the code base (both master and vNext) to conform to the agreed style ( variable as Type, right? :) )

Reply:
sure, sounds good.